src/Model/listmodel.js

```diff
@@ -1,7 +1,21 @@
 const Item = require('./itemmodel')
 
 class List extends Item {
+  constructor (title, active) {
```

Comment:
Ist auf jeden Fall ne Möglichkeit active beim erstellen manuell festzulegen. Denkbar wäre auch, dass eine neu erstellte Liste automatisch aktiv ist. Zum Beispiel ist es bei MS todo so, dass du eine neu erstellte Liste gleich angezeigt wird. 

D.h. für den constructor wäre es dann: `this.active = true` - Hierfür tendiere ich. Ist aber wie gesagt beides denkbar, entscheide du.

Reply:
Auf `this.active = true` angepasst.

Reply:
Habe es zu `this.active = true` geändert.